RELEASES.md

```diff
@@ -1,8 +1,22 @@
-# Release Process
-This document explains the process to release new version and to address patch release.
+# Releases
+This document explains the process to release new versions.
+
+## Versioning
+Versions are expressed as `vX.Y.Z` as well as Semantic Versioning;
```

Comment:
I don't think we're following this right? 🤔 I mean, we have not released any major yet, and minor does contain breaking changes. Directly mentioning that we're not follow the Semantic version maybe better.

Reply:
I meant that versions are "expressed" as well as SemVer, but it does not mean that we follow the rules of it. It may confuse readers. I will fix it 👍